public/locales/en/translation.json

```diff
@@ -49,6 +49,7 @@
     "edd-pfl-claim": "https://edd.ca.gov/Disability/How_to_File_a_PFL_Claim_in_SDI_Online.htm",
     "edd-filing-claim": "https://edd.ca.gov/Unemployment/Filing_a_Claim.htm",
     "edd-calculator": "https://www.edd.ca.gov/Unemployment/ui-calculator.htm",
+    "edd-cares": "https://edd.ca.gov/about_edd/coronavirus-2019/cares-act.htm",
```

Comment:
Should we add this to es/translation.json as well to keep the files in sync (i.e., the same number of entries in each json file)?